resources/domain/balance/index.ts

```diff
@@ -1,9 +1,17 @@
 import BigNumber from 'bignumber.js'
-import { NATIVE_CURRENCY } from '../../../../../resources/constants'
+import { NATIVE_CURRENCY } from '../../constants'
+
+interface DisplayedBalance extends Balance {
+  displayBalance: string
+  price: string
+  priceChange: string | false
+  totalValue: BigNumber
+  displayValue: string
+}
 
 const UNKNOWN = '?'
 
-export function formatBalance (balance, totalValue, decimals = 8) {
+export function formatBalance (balance:any, totalValue:BigNumber, decimals = 8) {
```

Comment:
can we get rid of this any? this is a `BigNumber` right?

Reply:
Yup, missed this one!